LCA_Analysis/LCA_analysis_2024_10_28.ipynb

```diff
@@ -0,0 +1,202 @@
+{
```

Comment:
Please add comments:)

Reply:
There are comments in the notebook, do you mean we need more?